README.md

```diff
@@ -40,15 +46,19 @@ Alternatively, you can use the `buildscript` DSL:
             maven {
                 url "https://nexus.gluonhq.com/nexus/content/repositories/releases"
             }
+            maven {
+                url "https://nexus.gluonhq.com/nexus/content/repositories/public-snapshots"
+            }
             maven {
                 url "https://plugins.gradle.org/m2/"
             }
         }
         dependencies {
-            classpath 'com.gluonhq:client-gradle-plugin:0.0.20'
+            classpath 'com.gluonhq:client-gradle-plugin:0.1.20-SNAPSHOT'
```

Comment:
Why does this has SNAPSHOT appended to the version?

Reply:
Right, should be release version